ingestion/functions/template.yaml

```diff
@@ -416,6 +416,42 @@ Resources:
         - AWSLambdaReadOnlyAccess
       Layers:
         - !Ref ParsingLibLayer
+  BrazilRioDeJaneiroParsingFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: parsing/brazil_rio_de_janeiro/
+      Handler: rio_de_janeiro.lambda_handler
+      Description: Parse case data for the Rio de Janeiro State of Brazil
+      # Function's execution role
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - AWSLambdaReadOnlyAccess
+      Layers:
+        - !Ref ParsingLibLayer
+  NewZealandParsingFunction:
```

Comment:
Looks like the checks are failing on New Zealand, since there isn't a parser up for it yet. Perhaps you can remove that for the time being?

Reply:
Good spot, thanks